pandas/_testing/__init__.py

```diff
@@ -923,7 +925,6 @@ def external_error_raised(expected_exception: type[Exception]) -> ContextManager
     Callable
         Regular `pytest.raises` function with `match` equal to `None`.
     """
-    import pytest
```

Comment:
why does this one need moving?

Reply:
Good question, pyupgrade removed it automatically. Lets move it back and see what happens